.gitignore

```diff
@@ -77,3 +77,8 @@ shellcheck-reports
 
 # Snyk
 .dccache
+
+# Konflux builds download scanner vuln files to the code repo.
+# These files need to be ignored in order for `make tag` return a clean version string.
+repository-to-cpe.json
+container-name-repos-map.json
```

Comment:
Sorry, I just realized now that these will sit in the repo root. As a suggestion, let's put them in some subdirectory to create a context of "scanner build time". I don't have a good directory to suggest but I may have a similar case to solve for `main`.

Reply:
da071cb